presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushAggregationThroughOuterJoin.java

```diff
@@ -299,10 +306,14 @@ private static boolean groupsOnAllColumns(AggregationNode node, List<Symbol> col
             }
 
             AggregationNode.Aggregation overNullAggregation = new AggregationNode.Aggregation(
-                    (FunctionCall) inlineSymbols(sourcesSymbolMapping, aggregation.getCall()),
                     aggregation.getFunctionHandle(),
+                    aggregation.getArguments().stream().map(argument -> inlineSymbols(sourcesSymbolMapping, argument)).collect(toImmutableList()),
+                    aggregation.getFilter().map(filter -> inlineSymbols(sourcesSymbolMapping, filter)),
+                    aggregation.getOrderBy(), // order by symbol should not be inlinable
```

Comment:
why is that?

Reply:
Yes, we should inline this as well. Will change it.